packages/expo-modules-core/ios/ReactDelegates/ExpoReactDelegate.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+/**
+ An extensible react instance creation delegate. This class will loop through each `ExpoReactDelegateHandler` to determine the winner to create the instance.
+ */
+@objc
+public class ExpoReactDelegate: NSObject {
+  private let handlers: [ExpoReactDelegateHandler]
+
+  public init(handlers: [ExpoReactDelegateHandler]) {
+    self.handlers = handlers
+  }
+
+  @objc
+  public func createBridge(delegate: RCTBridgeDelegate, launchOptions: [AnyHashable : Any]?) -> RCTBridge {
+    self.handlers.forEach { $0.bridgeWillCreate() }
+    let result = (self.handlers.lazy
+      .compactMap { $0.createBridge(reactDelegate: self, bridgeDelegate: delegate, launchOptions: launchOptions) }
+      .first(where: { _ in true }) ?? nil) ?? RCTBridge(delegate: delegate, launchOptions: launchOptions)!
+    self.handlers.forEach { $0.bridgeDidCreate(bridge: result) }
+    return result
+  }
+
+  @objc
+  public func createRootView(bridge: RCTBridge, moduleName: String, initialProperties: [AnyHashable : Any]?) -> RCTRootView {
+    return (self.handlers.lazy
+      .compactMap { $0.createRootView(reactDelegate: self, bridge: bridge, moduleName: moduleName, initialProperties: initialProperties) }
+      .first(where: { _ in true }) ?? nil) ?? RCTRootView(bridge: bridge, moduleName: moduleName, initialProperties: initialProperties)
```

Comment:
Isn't it the same? 😉 
```suggestion
    return self.handlers.lazy
      .compactMap { $0.createRootView(reactDelegate: self, bridge: bridge, moduleName: moduleName, initialProperties: initialProperties) }
      .first(where: { _ in true }) ?? RCTRootView(bridge: bridge, moduleName: moduleName, initialProperties: initialProperties)
```